src/Bridge/Doctrine/Orm/ItemDataProvider.php

```diff
@@ -64,11 +65,18 @@ public function getItem(string $resourceClass, $id, string $operationName = null
         }
 
         $manager = $this->managerRegistry->getManagerForClass($resourceClass);
+        $classMetadata = $manager->getClassMetadata($resourceClass);
+        $metadataIdentifier = $classMetadata->getIdentifier();
         if (null === $manager) {
             throw new ResourceClassNotSupportedException();
         }
 
-        $identifierValues = explode('-', $id);
+        $identifierValues = (array) $id;
```

Comment:
I'd be happy if this whole identifiers business can be moved into a separate method:

```php
$identifiers = $this->normalizeIdentifiers((array) $id);
```